contracts/SSVNetwork.sol

```diff
@@ -279,6 +280,8 @@ contract SSVNetwork is OwnableUpgradeable, ISSVNetwork {
             }
             // Changing to a single event reducing by 15K gas
         }
+
+        // WHY THIS EMIT IS NOT LAST THING OR AT LEAST AFTER VALIDATION
```

Comment:
@andrew-blox in solidity it doesn't matter - if the validation will be failed, the func flow will be reverted

Reply:
Yeah @lior-blox updated me as well on that, i personally still think its best to keep last just to follow same format as other functions.